homework-submission/week1-h.w/src/App.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from "react";
+import "./App.css";
+import Item from "./Item.js";
+import ItemData from "./ItemData.js";
+import Header from "./Header";
+
+class App extends Component {
+  render() {
+    const Data = ItemData.map((Data, Key) => (
+      <Item text={Data.text} deadline={Data.deadline} Key={Key} />
+    ));
```

Comment:
if you check your console you will fine the same error, that because you used Key with capital letter K in the Item tag.
you can use  capital letter as a parameter (it is your choice ) 
so it should be like:  key = { Key }
:-)

Reply:
correction is done. thank you!

Reply:
well done Joudi.....